src/Proxy.NET/Endpoints/Attendee.cs

```diff
@@ -0,0 +1,60 @@
+using System.Net;
+using System.Text.Json;
+using Proxy.NET.Models;
+using Proxy.NET.Services;
+
+namespace Proxy.NET.Endpoints;
+
+public static class Attendee
+{
+    private enum RequestType
+    {
+        AttendeeAdd,
+        AttendeeGetKey
+    }
+
+    public static void AttendeeEndpoints(this IEndpointRouteBuilder routes)
+    {
+        MapRoute.Post(routes, RequestType.AttendeeAdd, AttendeeAdd, Auth.Type.Jwt, "/attendee/event/{event_id}/register");
+        MapRoute.Get(routes, RequestType.AttendeeGetKey, AttendeeGetKey, Auth.Type.Jwt, "/attendee/event/{event_id}");
+    }
+
+    /// <summary>
+    /// Represents an asynchronous operation that can return a value.
+    /// </summary>
+    /// <returns>A task representing the asynchronous operation.</returns>
+    private static async Task AttendeeAdd(HttpContext context, RequestType requestType, string extPath)
+    {
+        var attendeeService = context.RequestServices.GetRequiredService<IAttendeeService>();
```

Comment:
You can use the [`[FromService]`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.fromservicesattribute?view=aspnetcore-8.0) attribute to bind a service as an argument to a route handler, rather than resolving it yourself.

Reply:
This will cause a signiture issue for the delegate that is used for methods that require different services to be injected

Reply:
We shouldn't be using a custom delegate. See the comments in the `MapRoute.cs` file. Essentially, you've recreated the API surface area of asp.net.